Bio/Cluster/__init__.py

```diff
@@ -13,11 +13,522 @@
 """
 
 import numpy
+import numbers
 
-from Bio.Cluster.cluster import version, kcluster, kmedoids, treecluster
-from Bio.Cluster.cluster import somcluster, clusterdistance
-from Bio.Cluster.cluster import clustercentroids, distancematrix, pca
-from Bio.Cluster.cluster import Node, Tree
+from . import _cluster
+
+__all__ = ['Node',
```

Comment:
Conventional to use a tuple rather than a list (some API tools complain about this otherwise)